src/net/NetworkServer.cpp

```diff
@@ -0,0 +1,124 @@
+#include "NetworkServer.h"
```

Comment:
Code duplicated between `Connection` and `NetworkClient`. Put a `Connection` inside `NetworkClient`? (And move `Connection` to its own file.)